### DISCUSSION TITLE
How to change port in executable

### DISCUSSION THREAD

**Discussion by pystar1007** [FlareSolverr/FlareSolverr#787] [General]
I wonder How to change port in executable. so I can run it several executable at a once. 

Is this Posssible?

**Comment by ngosang** [comment]
Set the environment variable PORT
https://github.com/FlareSolverr/FlareSolverr#environment-variables

**Comment by arashtotti88** [comment]
hi

i have the same problem 

please provide a step-by-step tutorial for amature exactly how to change port ?

i use FlareSolverr on ubuntu with docker and im struggling with charging port.

I tried to cahnge port from here and stopped and restarted the container but still can't change the port.

/var/lib/docker/containers/................../config.v2.json

looking forward for reply

thanks

**Reply by ilike2burnthing** [reply]
For Docker just change the port mapping, e.g. `-p 1234:8191` - https://github.com/FlareSolverr/FlareSolverr#docker

**Comment by arashtotti88** [comment]
i fix the problem with this code

sudo docker run -d \
  --name=flaresolverr \
  -p MYPORT:8191 \
  -e LOG_LEVEL=info \
  --restart unless-stopped \
  ghcr.io/flaresolverr/flaresolverr:latest

thank you so much
